jobs/JEVS_GLWU_PLOTS

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+#################################
+# SET SHELL PROCESSING VARIABLES 
+#################################
+set -x
+export PS4='+ $SECONDS + '
+date
+
+#########################################################
+# obtain unique LSF id (jobid) and make temp directories 
+#########################################################
+export DATA=${DATA:-${DATAROOT}/${jobid:?}}
+mkdir -p $DATA
+cd $DATA
+
+##########################################
+# Run setpdy and initialize PDY variables 
+##########################################
+export cycle=${cycle:-t00z}
+setpdy.sh 90
+. ./PDY
+export PDYm90=${PDYm90:-${PDYm90}}
+export VDATE=${VDATE:-${PDYm1}}
+
+####################################
+# Set data directives
+####################################
+export SENDECF=${SENDECF:-YES}
+export SENDCOM=${SENDCOM:-YES}
+export SENDDBN=${SENDDBN:-YES}
+
+####################################
+# Define NET/RUN variables          
+####################################
+export NET=${NET:-evs}
+export MODEL=${MODEL:-evs}
+export STEP=${STEP:-plots}
+export COMPONENT=${COMPONENT:-glwu}
+export MODELNAME=${MODELNAME:-glwu}
+export MODNAM=$(echo $MODELNAME | tr '[a-z]' '[A-Z]')
+export OBTYPE=${OBTYPE:-GDAS}
```

Comment:
I see that `OBTYPE` is set to GDAS in the glwu plots J-Job and is set to NDBC in the prep and stats J-Jobs. Should it be set to the same OBTYPE throughout?

@SamiraArdani-NOAA @malloryprow